web/client/components/catalog/RecordItem.jsx

```diff
@@ -210,6 +211,19 @@ class RecordItem extends React.Component {
                 </AddTMS>
             );
         }
+        if ( wfs ) {
```

Comment:
If I remember correcly, we talked of doing some cleanup refactoring of this stuff with the WFS work.

Reply:
Yes, you're right.
I had to fix many bugs on styler so I exhausted I wanted to dedicate to this refactor. Can we open a dedicated issue about this refactor?